docs/dg/dev/backend-development/opentelemetry/overview.md

```diff
@@ -0,0 +1,416 @@
+| `OTEL_BSP_SCHEDULE_DELAY`        | Defines a delay in ms that will be used to wait before sending batch of spans into the exporter. The bigger the number, the bigger the batch.                                | 1000                    | 0...100000000                                                  |
+| `OTEL_BSP_MAX_QUEUE_SIZE`        | Defines a max size of the queue of spans that can be processed in one request                                                                                                | 2048                    | at least an amount of spans you want to see                    |
+| `OTEL_BSP_MAX_EXPORT_BATCH_SIZE` | Defines a size of the batch of spans. After this size is reached, batch will be sent to the exporter.                                                                        | 512                     | more than 0 and less than `OTEL_BSP_MAX_QUEUE_SIZE`            |
+
+
+## Custom attributes
+
+You may want to add some custom data to your traces. E.g. a user id that was logged that time into your system. Or a current store name. Or anything else that can vary from request to request. In order to do so, you can add a custom attribute that will appear in the root span for better visibility.
+We would recommend to do so using `MonitoringService::addCustomParameter()` method. By default Spryker provides a few of those attributes already, like a current store name and locale.
+If you don't want to use `MonitoringService` for some reason, you may use `\Spryker\Service\Opentelemetry\OpentelemetryService::setCustomParameter` directly. It does exactly the same job, but without necessity to install `Monitoring` module.
+All attributes added via those services will be added into the root span on the very end of request execution, so you can call it even after response wes already sent.
+
+## Custom events
+
+If you want to add some custom event to you trace in order to configure some logic into your backend, you can trigger a custom event during the execution. It works pretty much the same as adding custom attributes, so it's pretty easy to use.
+The same as for custom attributes you can add those via `\Spryker\Service\Monitoring\MonitoringService::addCustomEvent()` or via `\Spryker\Service\Opentelemetry\OpentelemetryService::addEvent()` if you dont want to install `Monitoring` module.
+Event will be wired into the root span.
+
+## Error handling
+
+You are definitely want to track all possible errors during the execution. OOTB OpenTelemetry integration will catch all exceptions that were thrown during span execution. But there are possibility that error can be lost if part of your code was not covered with hooks, or spans were sampled out, or trace was marked as not detailed.
```

Comment:
```suggestion
You are definitely want to track all possible errors during the execution. OOTB OpenTelemetry integration will catch all exceptions that were thrown during span execution. But there is a possibility that error is lost if part of your code was not covered with hooks, or spans were sampled out, or trace was marked as not detailed.
```

Reply:
@gechetspr is it possible to have a configuration that makes sure that ALL exceptions are attached to the root span, no matter what.